js/templates/modals/moderatorDetails.html

```diff
@@ -57,13 +72,19 @@ <h4><%= ob.polyT('moderatorDetails.termsOfService') %></h4>
       </div>
       <div class="flexCol flexHCent gutterV">
         <% if (ob.cardState !== 'selected' && (!ob.ownMod || ob.purchase)) { %>
-          <button class="btn clrP clrBr js-addAsModerator">
-            <% if (ob.purchase) { %>
-              <%= ob.polyT('purchase.chooseModerator') %>
-            <% } else { %>
-              <%= ob.polyT('moderatorDetails.addAsModeratorButton') %>
-            <% } %>
-          </button>
+          <% if (ob.moderatorInfo.acceptedCurrencies.indexOf(ob.currencyMod.getServerCurrency().code) !== -1) { %>
```

Comment:
For this check and the one below, we can't just use getServerCurrency().code because that doesn't take testnet into account and we'll end up with no mods being available on testnet.

It would have something like:

```
ob.moderatorInfo.acceptedCurrencies.includes(
  ob.currencyMod.getServerCurrency().isTestnet ?   ob.currencyMod.getServerCurrency().testnetCode :   ob.currencyMod.getServerCurrency().code
)
```

Reply:
Good catch, thanks.